docs/topics/native/native-debugging.md

```diff
@@ -1,25 +1,50 @@
 [//]: # (title: Debugging Kotlin/Native)
 
-Currently, the Kotlin/Native compiler produces debug info compatible with the DWARF 2 specification, so modern debugger tools can
-perform the following operations:
-- breakpoints
-- stepping
-- inspection of type information
-- variable inspection
-
->Supporting the DWARF 2 specification means that the debugger tool recognizes Kotlin as C89, because before the DWARF 5 specification, there is no identifier for the Kotlin language type in specification.
+The Kotlin/Native compiler can generate binaries with debug info, as well as create debug symbol files for [symbolicating
+crash reports](#debug-ios-applications).
+
+The debug info is compatible with the DWARF 2 specification, so modern debugger tools, like LLDB and GDB can:
```

Comment:
Can we add a link to the DWARF specification somehow? Like https://dwarfstd.org/index.html ? 🤔